libs/coin-modules/coin-tron/src/test/cli.ts

```diff
@@ -53,7 +58,7 @@ function inferAccounts(account: Account, opts: Record<string, any>): AccountLike
     return accounts;
   }
 
-  return opts.token.map(token => {
+  return opts.token.map((token: string) => {
```

Comment:
Shouldn't we type `opts` instead ?

Reply:
It is the same mechanism for other coins. Frankly I don't know what is in `opts`. I wouldn't like to introduce here regression.